tests/e2e/tests/misc/lazy-module.ts

```diff
@@ -21,12 +22,42 @@ export default function() {
       if (oldNumberOfFiles >= currentNumberOfDistFiles) {
         throw new Error('A bundle for the lazy module was not created.');
       }
+      oldNumberOfFiles = currentNumberOfDistFiles;
+    })
+    // verify System.import still works
+    .then(() => writeFile('src/lazy-file.ts', ''))
+    .then(() => appendToFile('src/main.ts', `
+      // verify other System.import still work
+      declare var System: any;
+      const lazyFile = '-file';
+      System.import('./lazy' + lazyFile);
+    `))
+    .then(() => ng('build'))
+    .then(() => readdirSync('dist').length)
+    .then(currentNumberOfDistFiles => {
+      if (oldNumberOfFiles >= currentNumberOfDistFiles) {
+        throw new Error('A bundle for the lazy file was not created.');
+      }
+      oldNumberOfFiles = currentNumberOfDistFiles;
+    })
+    // verify 'import *' syntax doesn't break lazy modules
+    .then(() => npm('install', 'moment'))
+    .then(() => appendToFile('src/main.ts', `
+      import * as moment from 'moment';
+      console.log(moment);
+    `))
+    .then(() => ng('build'))
+    .then(() => readdirSync('dist').length)
+    .then(currentNumberOfDistFiles => {
+      if (oldNumberOfFiles >= currentNumberOfDistFiles) {
+        throw new Error('Bundles were not created after adding \'import *\'.');
+      }
     })
     // Check for AoT and lazy routes.
     .then(() => ng('build', '--aot'))
     .then(() => readdirSync('dist').length)
     .then(currentNumberOfDistFiles => {
-      if (oldNumberOfFiles >= currentNumberOfDistFiles) {
+      if (oldNumberOfFiles != currentNumberOfDistFiles) {
```

Comment:
The old number of files should be smaller, so I think this change is unneeded (and can lead to errors)?